extractor.go

```diff
@@ -460,6 +548,44 @@ func mapValidator(v abci.ValidatorUpdate) (*codec.Validator, error) {
 	}, nil
 }
 
+func mapValidators(v []*types.Validator) ([]*codec.Validator, error) {
+	validators := []*codec.Validator{}
+	if len(v) > 0 {
+		for _, v := range v {
+			valv := v
```

Comment:
You can rename the function argument to `srcValidators` (just an example) and avoid these variable name shenanigans altogether.